collectors/metadata/schemas/single-module.json

```diff
@@ -577,6 +613,37 @@
     "metrics"
   ],
   "$defs": {
+    "_categories": {
+      "type": "object",
+      "description": "Category object, with an entry per category depth.",
```

Comment:
What is category depth? Could you provide more information for people who don't follow all discussions like me?

Reply:
Will answer in second comment